source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace AdaptiveCards.Tools.IOSFeed
+{
+    public static class Constants
+    {
+        public const string ConnectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
+        public const string ContainerId = "adaptivecardsiosblobs";
+        public const string FrameworkPath = "source/ios/AdaptiveCards/AdaptiveCards";
+        public const string FrameworkName = "AdaptiveCards.framework.zip";
+        public const string PodspecPath = "./source/ios/tools/";
+        public const string TargetPodspecPath = "./source/ios/";
+        public const string PodspecName = "AdaptiveCards.podspec";
+    }
+
+    /// <summary>
+    /// IOSFeed picks up ios artifact, sends to the Azure storage 
+    /// and updates podspec for pod publication
+    /// </summary>
+    class IOSFeed
+    {
+        public static void Main()
+        {
+            // Run the examples asynchronously, wait for the results before proceeding
+            ProcessAsync().GetAwaiter().GetResult();
+        }
+
+        private static async Task ProcessAsync()
+        {
+            string storageConnectionString;
+
+            FileInfo fi = new FileInfo(Constants.ConnectionStringPath);
+            if (!fi.Exists)
+            {
+                throw new InvalidOperationException("connection string file does not exist");
```

Comment:
Nit: Helpful to include the path to what wasnt found for debugging? Where does this tool run again?

Reply:
This is part of external feed run by AzureDepOp, but it should be invoked by manual process since it's intended for external release. 

---
In reply to: [314076034](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r314076034) [](ancestors = 314076034)